lib/components/displays/NewContactForm.js

```diff
@@ -5,11 +5,15 @@ export default class NewContactForm extends Component {
   constructor(props){
     super(props);
     this.state = {
-      name: this.props.name || '',
-      email:this.props.email ||'',
-      socialFB:this.props.socialFB || '',
-      followUp:this.props.followUp || false,
-      notes:this.props.notes || ''
+      firstName: this.props.firstName || '',
+      lastName: this.props.lastName || '',
+      organization: this.props.organization || '',
+      email: this.props.email ||'',
+      socialFB: this.props.socialFB || '',
```

Comment:
@mziccardi Just a thought - what about putting social media stuff into ONE prop/state element as an object. Might be easier for us to manage the data as it starts to build out - i.e. we only have to pass one prop element and can pull the data out when we need it. 

Just a thought for additional development